Backend/routes/create-req.js

```diff
@@ -0,0 +1,27 @@
+const ans = require('../db/api_call')
+const helperConflict = require('../helpers/conflict')
+
+
+var createReq = async (req,res) => {
+    let temp = ""
+    temp = 'create/' + req.params.id
+    let overlaps = await helperConflict(req)
+    if ( !overlaps ) {

```

Comment:
consider the readability of what you are writing when naming variables, does this if statement make sense when you look at the message that is returned? Rename variables like this to make code more readable